.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Pull Request Template
+
+Please review the [Contribution Guidelines](https://github.com/sclorg/welcome/blob/master/contribution.md) before submitting the Pull Request.
+
+For more information about the Software Collection Organization, please visit the [Welcome](https://github.com/sclorg/welcome) pages.
+
+## Description
+
+Include here a summary of the change, what issue is fixed by it, and why is this change needed.
+List also the depencencies, if any exist.
+
+## Type of change
+
+Delete non-relevant options:
+
+- [ ] Bug fix
```

Comment:
Are the checkboxes useful if a user should delete non-relevant options? IIRC the checkboxes render as tasks in PR so the result of this will be either X completed task (if only the relevant ones are kept) or X out of Y completed tasks if all of them stay here. I think it's confusing either way.

Reply:
Good point. I was thinking, if it is even necessary to add here the 'type of change' section. What do you say if I remove it whole and leave only the description field?
Or I will ask today at the meeting.

Reply:
> Are the checkboxes useful if a user should delete non-relevant options? 

I have doublechecked and seems they are not. Thanks for catching this.